src/article_maker.cc

```diff
@@ -49,8 +49,10 @@ std::string ArticleMaker::makeHtmlHeader( QString const & word, QString const &
 
   // add jquery
   {
+    // TODO: document why Jquery cannot be removed or can we?
     result += R"(<script src="qrc:///scripts/jquery-3.6.0.slim.min.js"></script>)";
-    result += R"(<script> var $_$=jQuery.noConflict(); </script>)";
+    // TODO: document why noCliflict need to be called.
+    result += R"(<script> jQuery.noConflict(); </script>)";
```

Comment:
I have vague memory that including jQuery actually fixes some dictionaries that forget to include jQuery.

However, I cannot find the source of this claim and the `jQuery.noConflict()` seems to nullify this claim.

We need to document this fact here if jQuery cannot be removed. @xiaoyifang

Reply:
I remembered that some dictionaries will include their own version of JQuery which will conflict with the built-in JQuery.

Reply:
I think we have to keep it forever if it actually fixes something in the wild. Or someone implicitly have their own script invokes this jQuery.

I will just add comment about this reality.